frontend/src/js/App.js

```diff
@@ -43,9 +43,23 @@ let store = configureStore();
         $(this).on("DOMNodeInserted", child_inserted);
         return this;
 
-
     };
+}(jQuery));
 
+(function(jQuery) {
```

Comment:
What is this section doing? You seem to be writing a jquery plugin, but I'm not clear on why or what.